crypto/src/lib.rs

```diff
@@ -122,35 +118,13 @@ impl FromStr for Algorithm {
     allow(unused_tuple_struct_fields)
 )]
 #[derive(Debug, Clone)]
-enum KeyGenOption {
+pub enum KeyGenOption {
```

Comment:
```suggestion 
enum KeyGenOption {
```

I don't think it has to be `pub`